jekyll/_cci2/faq.md

```diff
@@ -10,6 +10,18 @@ order: 1
 * Contents
 {:toc}
 
+## What are the differences between CircleCI’s hosting options?
+
+- **Cloud** - CircleCI manages the setup, infrastructure, security and maintenance of your services. You get instant access to new feature releases and automatic upgrades, alleviating the need for manual work on an internal system.
+
+- **Server** - You install and manage CircleCI, through a service like AWS, behind a firewall that your team sets up and maintains according to your datacenter policy. You have full administrative control for complete customization and manage  upgrades as new versions are released.
```

Comment:
This seems to now be outdated since we're not using Cloud and Server and proper nouns

Reply:
this is what Cheryl asked me to document, so I will keep it for now and fix tomorrow if needed